api/src/main/java/com/codeforcommunity/api/IProtectedSiteProcessor.java

```diff
@@ -106,6 +108,21 @@ void uploadSiteImage(
    */
   List<FilterSitesResponse> filterSites(JWTData userData, FilterSitesRequest filterSitesRequest);
 
+  /**
+   * Retrieves unapproved site images, as detailed below
```

Comment:
Maybe we can rename the method to `filterUnapprovedSiteImages` so that in the future someone might not think that it is filtering from the set of all site images?